.vsts-pipelines/jobs/build-images.yml

```
@@ -20,6 +20,14 @@ jobs:
   variables:
     osVersion: ${{ parameters.osVersion }}
   steps:
+  - ${{ if eq(parameters.name, 'Build_Linux_arm32v7') }}:
+    - script: >
+        job="$(Agent.JobName)" &&
+        sampleFolder=$(echo "$job" | grep -o -E "aspnetapp|dotnetapp") &&
+        dockerfilePath="samples/$sampleFolder/Dockerfile" &&
+        sed 's/:2.2 AS/:2.2-bionic-arm32v7 AS/g' $dockerfilePath > Dockerfile.tmp &&
```

This should using the `2.2-stretch-slim-arm32v7` tag as that is what is included in the multi-arch tag previously used.
```suggestion
        sed 's/:2.2 AS/2.2-stretch-slim-arm32v7 AS/g' $dockerfilePath > Dockerfile.tmp &&
```